### PR TITLE
docs(scully): add FAQ on Why -- Are Required

### LINKED ISSUE #1358: Add an explanation on why -- is needed
### Description

Update the docs to indicate on why the `--` is needed when running Scully cli commands.

### PR DESCRIPTION
**Pull request by mainawycliffe**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [X] Other... Please describe: Docs Update

## What is the current behavior?

Add an FAQ entry to explain as to why the double hyphen -- is required when running `npm run scully` command.

Issue Number: #1358

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

Closes #1358

### CODE REVIEW

**Comment by mainawycliffe** [comment]
Closes #1358

File: docs/faq.md [hunk 1]
```diff
@@ -250,3 +250,14 @@ to:
 ```
 
 </details>
+
+## Scully Command Line Interface
+
+<details>
+<summary>Why are the pair of hyphens - `--` - used when running `yarn scully`?</summary>
```
Comment (SanderElias) [inline docs/faq.md:257]: yarn doesn't have this issue, its strictly NPM.

**Review by SanderElias — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 1 above)
Comment (mainawycliffe) [reply docs/faq.md:257]: Ah, I see. Will make changes accordingly.

**Review by SanderElias — APPROVED** [review]

File: docs/faq.md [hunk 2]
```diff
@@ -257,3 +260,14 @@ to:
 ```
 
 </details>
+
+## Scully Command Line Interface
+
+<details>
+<summary>Why are the pair of hyphens - `--` - used when running `npm run scully`?</summary>
```
Comment (Jefiozie) [inline docs/faq.md:267]:
```suggestion
<summary>Why are the pair of hyphens - `--` - used when running `npm run Scully`?</summary>
```

File: docs/faq.md [hunk 3]
```diff
@@ -257,3 +260,14 @@ to:
...
+
+The pair of hyphens i.e. `--` indicate to node js that this is the end of node
```
Comment (Jefiozie) [inline docs/faq.md:269]:
```suggestion
The pair of hyphens i.e. `--` indicates to node js that this is the end of node
```

**Review by Jefiozie — COMMENTED** [review]
Hey @mainawycliffe, thank you for making these changes, we appreciate it.
I have added two small suggestions, it is more a grammatical thing, would you mind changing it?

**Comment by mainawycliffe** [comment]
@Jefiozie I have committed the suggestions, they make sense.

**Review by SanderElias — APPROVED** [review]

**Pull request merged**